CDCgov/NEDSS-Modernization
# CNFT2-1425-numeric-special-characters-fix-page-name

## Description

Entering a full string of numbers and special characters as a Page name are accepted when clicking Create Page.

## Tickets

* [CNFT2-1425](https://cdc-nbs.atlassian.net/browse/CNFT2-1425)

## Steps to verify

1. Navigate to /page-builder/manage/pages
2. Then click on Create new page
3. Scroll down to page name and verify only some special character work and not all


[CNFT2-1425]: https://cdc-nbs.atlassian.net/browse/CNFT2-1425?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
